tensorboard/compat/proto/BUILD

```diff
@@ -26,24 +26,32 @@ tb_proto_library(
         "meta_graph.proto",
         "node_def.proto",
         "op_def.proto",
-        "resource_handle.proto",
         "rewriter_config.proto",
         "saved_object_graph.proto",
         "saver.proto",
         "step_stats.proto",
         "struct.proto",
         "summary.proto",
-        "tensor.proto",
         "tensor_description.proto",
-        "tensor_shape.proto",
         "tfprof_log.proto",
         "trackable_object_graph.proto",
-        "types.proto",
         "variable.proto",
         "verifier_config.proto",
         "versions.proto",
     ],
     visibility = ["//visibility:public"],
+    deps = [":tensor_proto_lib"],
```

Comment:
Rather than using deps here, let's just leave the files in this rule in addition to the tensor_proto_lib rule - it should be okay for them to be included in more than one tb_proto_library.  Otherwise it's kind of odd that protos_all needs to depend on `tensor_proto_lib` even for things that it uses directly, not just via TensorProto.

Reply:
Yeah that's what I was doing initially, but the Travis CI didn't like it, saying resource_handle.py is already generated by "protos_all".

Reply:
... to handle the oddity properly, it would seem like we need to tear apart protos_all, define separate proto libs, and build the dependency graph.

Reply:
Ah my bad, you're right.  Never mind on this one then, sorry.